app/src/main/java/edu/nextstep/camp/calculator/MainActivity.kt

```diff
@@ -77,4 +82,12 @@ class MainActivity : AppCompatActivity(), MainContract.View {
         }
         Toast.makeText(this, toastMessage, Toast.LENGTH_SHORT).show()
     }
+
+
+    override fun showCalculateHistory(history: List<String>) {
+        val builder = AlertDialog.Builder(this).apply {
+            setMessage(history.joinToString(separator = "\n\n"))
+        }
+        builder.create().show()
```

Comment:
```suggestion
        AlertDialog.Builder(this)
            .apply { setMessage(history.joinToString(separator = "\n\n")) }
            .create()
            .show()
```
함수 체이닝을 활용하면, 별도의 변수를 정의하지 않아도 되요.